content/en/docs/ibm/deploy/install-kubeflow-on-iks.md

```diff
@@ -214,9 +236,40 @@ Check the pod `authservice-0` is in running state in namespace `istio-system`:
 kubectl get pod authservice-0 -n istio-system
 ```
 
-## Next steps
+### Extra network setup requirement for `vpc-gen2` clusters only
+
+**Note**: These steps are not required for `classic` clusters, i.e. where `WORKER_NODE_PROVIDER` is set to `classic`.
+
+A `vpc-gen2` cluster does not assign a public IP address to the Kubernetes master node by default.
+It provides access via a Load Balancer, which is configured to allow only a set of ports over public internet.
+Access the cluster's resources in a `vpc-gen2` cluster, using one of the following options,
 
-To secure the Kubeflow dashboard with HTTPS, follow the steps in [Exposing the Kubeflow dashboard with DNS and TLS termination](../authentication/#exposing-the-kubeflow-dashboard-with-dns-and-tls-termination).
+* Load Balancer method: To configure via a Load Balancer, go to [Expose the Kubeflow endpoint as a LoadBalancer](#expose-the-kubeflow-endpoint-as-loadbalancer).
+    This method is recommended when [Multi-user, auth-enabled](#multi-user-auth-enabled) is setup, otherwise it will expose
```

Comment:
```suggestion
    This method is recommended when you have Kubeflow deployed with [Multi-user, auth-enabled](#multi-user-auth-enabled) support — otherwise it will expose
```